core/body/body.ctrl.js

```diff
@@ -64,9 +64,16 @@ export class BodyCtrl extends View {
 			const pathFinder = new PathService(this.bag.body);
 			const cell = pathFinder.cell(e.path);
 
-			const editMode = this.model.edit().mode;
+			const edit = this.model.edit();
+			const editMode = edit.mode;
+			const editMethod = edit.method;
 			if (selectionState.mode === 'range') {
```

Comment:
can we have `selectionState.mode === 'range' || (editMethod === 'batch' && edit.state === 'startBatch')`?